examples/experimental/i3s-17-and-debug/helpers/build-minimap-data.js

```diff
@@ -0,0 +1,25 @@
+import {Vector3} from '@math.gl/core';
+import {Ellipsoid} from '@math.gl/geospatial';
+import {OrientedBoundingBox} from '@math.gl/culling';
+
+export function buildMinimapData(tiles) {
+  return tiles
+    .map(tile => {
+      if (!tile.selected || !tile.viewportIds.includes('main')) {
+        return null;
+      }
+      const boundingVolume = tile.boundingVolume;
+      const cartographicOrigin = new Vector3();
+      Ellipsoid.WGS84.cartesianToCartographic(boundingVolume.center, cartographicOrigin);
+      let radius = boundingVolume.radius;
+      if (!radius && boundingVolume instanceof OrientedBoundingBox) {
+        const halfSize = boundingVolume.halfSize;
+        radius = new Vector3(halfSize[0], halfSize[1], halfSize[2]).len();
```

Comment:
Isn't there a method to get a BoundingSphere from an OBB so that we don't have to duplicate such logic?